### PR TITLE
Added udev-rules

### PR DESCRIPTION
**Pull request by grigorovici-andreea**
Added udev-rules for eeprom access, for Dragonboard, Raspberrypi, Jetson.

Signed-off-by: Andreea Grigorovici <andreea.grigorovici@analog.com>

Solves #235

### CODE REVIEW

File: 53-aditofsdk.rules [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# allow read/write access to the eeprom
```
Comment (DanielGuramulta) [inline 53-aditofsdk.rules]: I think we can move this file in another place, maybe utils or misc, it shouldn't be in the top level directory

File: CMakeLists.txt [hunk 2]
```diff
@@ -50,6 +50,17 @@ if (WITH_ROS)
         add_subdirectory(bindings/ros)
 endif()
 
+# install udev rules
```
Comment (DanielGuramulta) [inline CMakeLists.txt]: Could group this commands into a cmake file that can be stored in the `cmake` folder, and just include it here with a comment line ######################### Udev Rules ####################### or something to match the rest of the comments in this file

File: scripts/dragonboard410c/setup.sh [hunk 3]
```diff
@@ -21,6 +21,8 @@ print_help() {
         echo "        Build and install the minimum required version of opencv for the dnn example (3.4.1)"
         echo "-op|--opencvpath"
         echo "        Path to opencv installation "
+		echo "-ur|--udevrules"
```
Comment (DanielGuramulta) [inline scripts/dragonboard410c/setup.sh]: For all the scripts, formatting looks a bit odd. To many spaces/tabs in some cases

File: 53-aditofsdk.rules (hunk 1 above)
Comment (dNechita) [inline 53-aditofsdk.rules]: it's also the temp sensor

File: 53-aditofsdk.rules [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+SUBSYSTEM=="i2c", 
+RUN+="/bin/chown root:i2c /sys/bus/i2c/devices/0-0056/eeprom", 
+RUN+="/bin/chmod 0666 /sys/bus/i2c/devices/0-0056/eeprom", 
+RUN+="/bin/chown root:i2c /sys/bus/i2c/devices/1-0056/eeprom", 
+RUN+="/bin/chmod 0666 /sys/bus/i2c/devices/1-0056/eeprom" ,
+RUN+="/bin/chown root:i2c /sys/bus/i2c/devices/6-0056/eeprom", 
+RUN+="/bin/chmod 0666 /sys/bus/i2c/devices/6-0056/eeprom" ,
+RUN+="/bin/chmod 666 /dev/i2c-1" ,
+RUN+="/bin/chmod 666 /dev/i2c-0",
+RUN+="/bin/chmod 666 /dev/i2c-6" ,
```
Comment (dNechita) [inline 53-aditofsdk.rules]: I thought that you'll do a udev_rules file for each board that we support. Is this a combination for all of them?

Comment (grigorovici-andreea) [reply 53-aditofsdk.rules]: yes, this is a combination. Should I separate them?

Comment (dNechita) [reply 53-aditofsdk.rules]: yes

File: 53-aditofsdk.rules (hunk 1 above)
Comment (grigorovici-andreea) [reply 53-aditofsdk.rules]: moved to utils

File: CMakeLists.txt (hunk 2 above)
Comment (grigorovici-andreea) [reply CMakeLists.txt]: solved. thanks

File: scripts/dragonboard410c/setup.sh (hunk 3 above)
Comment (grigorovici-andreea) [reply scripts/dragonboard410c/setup.sh]: solved. thanks

File: 53-aditofsdk.rules (hunk 4 above)
Comment (grigorovici-andreea) [reply 53-aditofsdk.rules]: solved. thanks

File: 53-aditofsdk.rules (hunk 1 above)
Comment (grigorovici-andreea) [reply 53-aditofsdk.rules]: solved.

**Pull request merged**